dso/model.py

```diff
@@ -536,6 +545,90 @@ def purge_old_states(
 
             self.state.remove(state)
 
+# Falco
+
+@dataclasses.dataclass(frozen=True)
+class FalcoEvent:
+    message: str
+    cluster: str
+    hostname: str
+    time: datetime.datetime
+    rule: str
+    priority: str
+    output: dict
+
+@dataclasses.dataclass(frozen=True)
+class ExceptionTemplate:
+    template: str
+
+@dataclasses.dataclass(frozen=True)
+class ClusterNode:
+    cluster: str
+    node: str
+    count: int
+
+@dataclasses.dataclass(frozen=True)
+class FalcoEventGroup(Finding):
+    """
+    FalcoEventGroup represents a group of Falco events that are similar in
+    nature. In almost all cases those are false posities and can be ignored.
+    Falco exceptions can be defined but they can be silenced here.
+    """
+    rule: str
+    message: str
+    nodes: list[ClusterNode]
+    landscape: str
+    project: str
+    priority: str
+    """Falco priority, one of EMERGENCY, ALERT, CRITICAL, ERROR, WARNING,
```

Comment:
introduce `StrEnum` for falco priorities

Reply:
done